clamav_versions.py

```
@@ -0,0 +1,57 @@
+import dns.resolver
+import logging
+import sys
+
+logger = logging.getLogger("CLAMAV-REST.VERSIONS_SERVICE")
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+'''
+Kudos to https://forums.contribs.org/index.php?topic=49721.0 
+'''
+
+
+def split_version(version_text, divider, offset):
+    if not version_text:
+        raise Exception("version_text is required")
+
+    split = version_text.split(divider)
+
+    if len(split) < offset + 1:
```

Why not just let it blow up if the index doesn't' exist? EAFTP

 e.g. 
```
return split[offset]
```

or if you need to catch the exception: 

```
try:
    return split[offset]
except IndexError:
    raise .. 
```